api/src/cms/dto/contentType.dto.ts

```diff
@@ -34,11 +35,11 @@ export class FieldType {
   label: string;
 
   @IsString()
-  @IsEnum(['text', 'url', 'textarea', 'checkbox', 'file', 'html'], {
+  @IsEnum(FieldType, {
```

Comment:
```suggestion
 @IsNotEmpty()
  @IsEnum(FieldType, {
```

Hi @IkbelTalebHssan, do we need maybe to add  the @**IsNotEmpty** decorator to be sure that this field is not **undefined** or **null**

Reply:
Hi @yassinedorbozgithub Thanks for the review and the suggested changes :pray: 
Actually from the UI we can't pass an empty type for a field but this still be a good practice.
Changes are applied :heavy_check_mark: